src/v2/view-builder/components/Link.js

```diff
@@ -23,7 +23,8 @@ const Link = View.extend({
     let href = this.options.href || '#';
     return {
       'data-se': this.options.name,
-      href: href
+      href: href,
+      target: this.options.target,
```

Comment:
in case when `this.options.target` doesn't exist, should it default to `_self`? `target: this.options.target || '_self'`

Reply:
I don't think we need to do this because it will not add the target to the element if there is no value ({...target: undefined}), which will open the window self, we don't need to explicitly add '_self'.